stacks/observability/jaeger.yaml

```diff
@@ -14,15 +14,8 @@ options:
     cassandra: false
   allInOne:
     enabled: true
-    extraEnv: []
-    service:
-      headless: true
-      collector:
-        otlp:
-          # http:
-          #   name: otlp-http
-          grpc:
-            name: otlp-grpc
+  storage:
+    type: memory
```

Comment:
From memory, this cause me problem as soon as enough traces came in.
I think we should go back to disk storage (doesn't have to be a PVC or anything, in the pod is fine).

Reply:
What of the following would it be then?
https://github.com/jaegertracing/helm-charts/blob/9c9abf46461612ef3f2a11149200a45b35c9a82c/charts/jaeger/values.yaml#L100C3-L100C75

Reply:
I recall you mentioning something about Cassandra but it isn't mentioned in this PR.

Reply:
badger appears to be the light-weight _on-disk_ option, however it isn't default: https://github.com/jaegertracing/helm-charts/blob/05e6fa34d57820bf7eed997e32de9968c5e62b41/charts/jaeger/values.yaml#L179-L184

So I'm happy to accept your change and I will do something about moving to badger later.

Reply:
I will leave it up to you if you want to try `badger` (could be a rabbit hole), or leave as is. The change is approved.